react/wbooks/src/app/screens/Home/layout.js

```diff
@@ -1,20 +1,28 @@
 import React from 'react';
 import { array } from 'prop-types';
+import cn from 'classnames';
+import { t } from 'i18next';
 
 import NavBar from '../../components/NavBar';
 import BookInfo from '../../components/BookInfo';
 
 import styles from './styles.module.scss';
 
 function Home({ books }) {
+  const areBooks = books.length > 0;
+
   return (
-    <div className={styles.container}>
+    <div className={cn({ 'background-wild-sand': areBooks, 'background-white': !areBooks })}>
       <NavBar />
-      <div className={`row ${styles.booksContainer}`}>
-        {books.map(({ author, title, id }) => (
-          <BookInfo author={author} title={title} className="m-right-7 m-top-7" key={id} />
-        ))}
-      </div>
+      {areBooks > 0 ? (
+        <div className={`row ${styles.booksContainer}`}>
+          {books.map(({ author, title, id }) => (
+            <BookInfo author={author} title={title} className="m-right-7 m-top-7" key={id} />
```

Comment:
avoid using top margins, because the order is from top to bottom, so you should separate the elements from bottom margins. Someday you will have a problem with layouts because of this practice.

Reply:
Great